test/safe/discovery.test.ts

```diff
@@ -43,11 +88,17 @@ describe("Discovery Queue Test", () => {
 	});
 
 	const mockGitHubReposResponses = () => {
-		githubAccessTokenNock(1234);
+		githubAccessTokenNock(TEST_INSTALLATION_ID);
 
-		githubNock.get("/installation/repositories?per_page=100")
-			// eslint-disable-next-line @typescript-eslint/no-var-requires
-			.reply(200, require("../fixtures/list-repositories.json"));
+		if (useNewGithubClient) {
+			githubNock
+				.post("/graphql")
+				.reply(200, mockGitHubClientResponse);
+		} else {
+			githubNock.get("/installation/repositories?per_page=100")
+				// eslint-disable-next-line @typescript-eslint/no-var-requires
+				.reply(200, require("../fixtures/list-repositories.json"));
+		}
```

Comment:
/super-nit It will make sense to have 2 separate tests. 1 feature flag ON, another OFF.
Because we have too much if/else branches within our tests